app/templates/components/billing/payment-details-tab.hbs

```diff
@@ -249,9 +249,9 @@
         <p class="invoice-issue-banner invoice-issue-banner-no-invoices" data-test-help-text='true'>
           <SvgImage @name='icon-help' @class='icon-help' />
           Having trouble with your invoices?
-          <LinkTo @route='help' @query={{hash anchor='form' billing=true}}>
+          <ExternalLinkTo @href={{config-get 'urls.resources'}} @title="Travis CI Resources">
```

Comment:
this is what's rendered here (no urls.resources defined):
<a title="Travis CI Resources" rel="noopener noreferrer" target="_blank" id="ember644" class="ember-view">  
        We’re happy to help
      
</a>

Reply:
Added missing part - https://github.com/travis-ci/travis-web/pull/2922/commits/09d17216c9f7b78cc5d50738d6b9448639b8f8b5

Reply:
<img width="509" alt="image" src="https://github.com/user-attachments/assets/aeeb88d6-43fa-4e0e-8170-7adb1953d1c4">